src/App.js

```diff
@@ -1,8 +1,26 @@
-import React from 'react';
+import { Link, Route, Routes } from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Home from './components/Home';
+import Quote from './components/Quotes';
 
 const App = () => (
-  <Calculator />
+  <main>
+    <nav className="navigation">
+      <h1>Math Magicians</h1>
+      <ul>
+        <Link to="/">Home</Link>
+        <Link to="/calculator">Calculator</Link>
+        <Link to="/quote">Quote</Link>
+      </ul>
+      <hr />
+    </nav>
```

Comment:
- [ ] From the project requirement, the general layout of this app should match the given wireframes in the [project requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md). But comparing the given wireframes to your project layout, your navigation bar does not match the given wireframe. As you can see below, the `Math Magicians` text should lie on the right and the navigation links should lie on the left on the same line.
### Your layout
![Screenshot from 2023-01-26 07-51-54](https://user-images.githubusercontent.com/92303891/214783544-7ab0f1d6-9670-4def-a3ff-41025d335ffb.png)

### Wireframe
![Screenshot from 2023-01-26 07-44-05](https://user-images.githubusercontent.com/92303891/214783403-58f788a3-c32f-41d7-be71-9145cf648f6c.png)